lib/src/options.dart

```diff
@@ -219,7 +223,8 @@ class VideoPublishOptions {
     List<VideoParameters>? videoSimulcastLayers,
     List<VideoParameters>? screenShareSimulcastLayers,
     String? videoCodec,
-    BackupVideoCodec? backupCodec,
+    bool? enableBackupCodec,
```

Comment:
I think we ended up settling for having this inside `BackupVideoCodec` type?

Reply:
Removed `bool? enableBackupCodec`, only use `nullable backupVideoCodec`